lib/crowbar/client/config.rb

```diff
@@ -63,6 +62,50 @@ def values
 
       protected
 
+      def default_alias
+        ENV["CROWBAR_ALIAS"] || "default"
+      end
+
+      def default_username
+        ENV["CROWBAR_USERNAME"] || "crowbar"
+      end
+
+      def default_password
+        ENV["CROWBAR_PASSWORD"] || "crowbar"
+      end
+
+      def default_server
+        ENV["CROWBAR_SERVER"] || "http://127.0.0.1:80"
+      end
+
+      def default_timeout
+        if ENV["CROWBAR_TIMEOUT"].present?
+          ENV["CROWBAR_TIMEOUT"].to_i
+        else
+          60
+        end
+      end
+
+      def default_anonymous
+        if ENV["CROWBAR_ANONYMOUS"].present?
+          [
+            true, 1, "1", "t", "T", "true", "TRUE"
```

Comment:
It depends, if it's set through ruby it can be something different. That way we are always on the safe side :)